db/db/migrations/20230307094014_orders_item.sql

```diff
@@ -0,0 +1,16 @@
+-- migrate:up
+CREATE TABLE orders_item (
+  id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+  quantity INT NOT NULL,
+  products_option_id INT NOT NULL,
+  orders_id INT NOT NULL,
+  orders_status_id INT NOT NULL,
```

Comment:
```suggestion
  order_id INT NOT NULL,
  order_status_id INT NOT NULL,
```
밑의 `CONSTRAINT` 제약 부분도 전부 수정해주세요